src/Microsoft.ML.Data/Transforms/DropColumnsTransform.cs

```diff
@@ -237,6 +237,16 @@ private static VersionInfo GetVersionInfo()
         private const string DropRegistrationName = "DropColumns";
         private const string KeepRegistrationName = "KeepColumns";
 
+        public DropColumnsTransform CreateColumnDroper(IHostEnvironment env, IDataView input, params string[] columnsToDrop)
+        {
+            return new DropColumnsTransform(env, new Arguments() { Column = columnsToDrop }, input);
+        }
+
+        public static DropColumnsTransform CreateColumnSelector(IHostEnvironment env, IDataView input, params string[] columnsToKeep)
+        {
+            return new DropColumnsTransform(env, new KeepArguments() { Column = columnsToKeep }, input);
+        }
```

Comment:
In order to *keep* columns, we do `DropColumnsTransform.CreateColumnSelector`. :) The way that we resolve this on the command line, entry-points, the GUI, and whatnot, is we have this "KeepColumnsTransform" loadname declared, to make it *look* like there is another transform.

The fact that we do so in the underlying C# code is just because previously this little bit of confusion didn't matter, since we weren't advocating that people use this directly. However since we now are, we ought to adjust accordingly.

For that reason, I wonder if we can make the relationship explicit...

```csharp
public static class KeepColumnsTransform
{
    public IDataTransform Create(IHostEnvironment env, IDataView input, params string[] columnsToKeep)
        => new DropColumnsTransform(env, new KeepArguments() { Column = columnsToKeep }, input);
}
```

or something along these lines.

Once that is done, we can change "drop" creator back to a plain old constructor.